### ISSUE TITLE
Failed on v.qq.com

### ISSUE THREAD

**Issue by asakura2020** [soimort/you-get#1778]
Please make sure these boxes are checked before submitting your issue – thank you!

- [/ ] You can actually watch the video in your browser or mobile application, but not download them with `you-get`.
- [ /] Your `you-get` is up-to-date.
- [/ ] I have read <https://github.com/soimort/you-get/wiki/FAQ> and tried to do so.
- [/ ] The issue is not yet reported on <https://github.com/soimort/you-get/issues> or  <https://github.com/soimort/you-get/wiki/Known-Bugs>. If so, please add your comments under the existing issue.
- [ /] The issue (or question) is really about `you-get`, not about some other code or project.

Run the command with the `--debug` option, and paste the full output inside the fences:

```

C:\WINDOWS\system32>you-get -d https://v.qq.com/x/page/t0016gu6eyq.html
you-get: version 0.4.652, a tiny downloader that scrapes the web.
you-get: ['https://v.qq.com/x/page/t0016gu6eyq.html']
Traceback (most recent call last):
  File "<string>", line 11, in <module>
  File "src\you_get\__main__.py", line 92, in main
  File "src\you_get\common.py", line 1407, in main
  File "src\you_get\common.py", line 1320, in script_main
  File "src\you_get\common.py", line 1136, in download_main
  File "src\you_get\common.py", line 1400, in any_download
  File "src\you_get\extractors\qq.py", line 111, in qq_download
  File "src\you_get\extractors\qq.py", line 55, in qq_download_by_vid
UnboundLocalError: local variable 'ext' referenced before assignment
you-get returned -1

```

If there's anything else you would like to say (e.g. in case your issue is not about downloading a specific video; it might as well be a general discussion or proposal for a new feature), fill in the box below; otherwise, you may want to post an emoji or meme instead:

> Not able to download from this page although some may work, the failed one tend to point at this video only

C:\WINDOWS\system32>you-get -i https://v.qq.com/x/page/failed_example
Site:       QQ.com
Title:      《有完没完》动人MV 王啸坤再唱一无所有
Type:       MPEG-4 video (video/mp4)
Size:       87.88 MiB (92145718 Bytes)

> [OR HAVE SOME :icecream:!]

汉语翻译最终日期：2016年02月26日

在提交前，请确保您已经检查了以下内容!

- [x] 你可以在浏览器或移动端中观看视频，但不能使用`you-get`下载.
- [x] 您的`you-get`为最新版.
- [x] 我已经阅读并按 <https://github.com/soimort/you-get/wiki/FAQ> 中的指引进行了操作.
- [x] 您的问题没有在<https://github.com/soimort/you-get/issues> , <https://github.com/soimort/you-get/wiki/FAQ> 或 <https://github.com/soimort/you-get/wiki/Known-Bugs> 报告，否则请在原有issue下报告.
- [x] 本问题确实关于`you-get`, 而不是其他项目.

请使用`--debug`运行，并将输出粘贴在下面:

```
[在这里粘贴完整日志]
```

如果您有其他附言，例如问题只在某个视频发生，或者是一般性讨论或者提出新功能，请在下面添加;或者您可以卖个萌:

> [您的内容]
> [舔 :icecream:!]

**Comment by omaidb** [comment]
v.qq.com 下载速度好慢，只有30k，这是为什么

**Comment by cnbeining** [comment]
@omaidb 限速。

**Comment by soimort** [comment]
Closed by #1785.
